UserInterface/MenuItemDemos/README.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Xamarin.Forms - MenuItem Demos
+description: "The MenuItem control is used to define context menu choices for ListView items."
```

Comment:
Technically MenuItem isn't a control (ditto for ToolbarItem). All our controls are listed at https://docs.microsoft.com/xamarin/xamarin-forms/user-interface/controls/ and MenuItem (and ToolbarItem) deliberately aren't listed there.

Better to refer to it as the MenuItem class (also update the ToolbarItem readme/doc).

Reply:
Also, the MenuItem class is used in Shell applications for flyout menu items, so it's not for context menus in a ListView.